interfaces/fiveg_core_gnb/v0/schema.py

```diff
@@ -0,0 +1,57 @@
+"""This file defines the schemas for the provider and requirer sides of the `fiveg_core_gnb` relation interface.
+
+It must expose two interfaces.schema_base.DataBagSchema subclasses called:
+- ProviderSchema
+- RequirerSchema
+
+Examples:
+    ProviderSchema:
+        unit: <empty>
+        app: {
+            "tac": 1,
+            "plmns": [
+                {
+                    "mcc": "001",
+                    "mnc": "01",
+                    "sst": 1,
+                    "sd": 1,
+                }
+            ],
+        }
+    RequirerSchema:
+        unit: <empty>
+        app: {
+            "cu_id": "gnb001",
```

Comment:
```suggestion
            "cu_name": "gnb001",
```

I think this is GNB name instead of ID.